Builder.cs

```diff
@@ -449,6 +449,8 @@ private void WriteItems()
                 }
             }
 
+            SceneUtils.WriteShuffledDungeonChests(new int[][] { ActorUtils.Permutation(8), ActorUtils.Permutation(14), ActorUtils.Permutation(11), ActorUtils.Permutation(24) });
```

Comment:
Possibly the biggest flaw in the design of this feature, but I guess it depends on how everyone else wants to design things.  As I see it though, in MMR the Randomizer class is responsible for shuffling any randomized elements, while the Builder class is suppose to take the result of that and actually construct the rom. The key thing here is that the Randomizer class functions without having a base rom, which allows people to generate a spoiler without having one.

Reply:
this is going to be changed to construct itself based on the item list being passed down from _randomized. Another temporary and disgusting hack. It felt awful to write.